packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1165,6 +1174,14 @@ class SemanticsProperties extends DiagnosticableTree {
   /// [maxValueLength] is set.
   final int? currentValueLength;
 
+  /// Provides an identifier for UI testing. This value is not exposed to the
```

Comment:
Can you use doc template to reuse the doc in SemanticsData and SemanticsProperties?

Reply:
Also this will not only be used by UI testing? I think every tool that uses UIAutomator can use this?

Reply:
> Also this will not only be used by UI testing? I think every tool that uses UIAutomator can use this?

That's right. UI testing is just the most common use case.

Reply:
> Can you use doc template to reuse the doc in SemanticsData and SemanticsProperties?

How can I do that?

Reply:
done!